stories/utils/generate-widget-stories.js

```diff
@@ -210,24 +208,20 @@ export function generateReportBasedWidgetStories( {
 		...customVariants,
 	};
 
-	let widget;
+	let widgetElement;
+
 	if ( wrapWidget ) {
-		const slugs = moduleSlugs.map( ( slug ) => {
-			return `${ slug }-widget`;
-		} );
-		widget = (
-			<Widget slug={ slugs.join( ' ' ) }>
-				<Component />
-			</Widget>
-		);
+		widgetElement = <Component />;
```

Comment:
@benbowler storybook is broken again. See data unavailable states of widgets: 
* https://google.github.io/site-kit-wp/storybook/pull/2676/?path=/story/analytics-module-components-dashboard-bounce-rate-widget--data-unavailable
* https://google.github.io/site-kit-wp/storybook/pull/2676/?path=/story/analytics-module-components-dashboard-goals-widget--data-unavailable

I think it happens because this `<Component />` should be wrapped with a `Widget` element and receive additional props too:

```javascript
let widgetElement;

const slug = moduleSlugs.map( ( mapSlug ) => `${ mapSlug }-widget` ).join( ' ' );
const widgetComponentProps = getWidgetComponentProps( slug );

if ( wrapWidget ) {
    const { Widget } = widgetComponentProps;
    widgetElement = (
        <Widget>
            <Component />
        </Widget>
    );
} else {
    widgetElement = <Component { ...widgetComponentProps } />;
}
```

Reply:
Agreed - we must not remove the `Widget` wrapper here: The widget is registered with `wrapWidget: true`, so the `Widget` needs to be present _around_ the registered `Component`.

One thing that would need to be fixed in @eugene-manuilov's code snippet above: We would still need to pass `...widgetComponentProps` to the `Component` either way, since that's expected. Even if the component doesn't use `props.Widget`, it might use other components on there, e.g. `props.WidgetReportZero`.

Reply:
Ah... yes, thanks! I mentioned it in the comment, but forgot to add it to the snipped... :smile:

Reply:
Ok, thanks for updating the comment, I've applied that and will push shortly.